package repaso;

public class EmpleadoFjo extends Empleado{
	int horas;

	public EmpleadoFjo(String nombre, int horas) {
		super(nombre);
		this.horas = horas;
	}
	@Override
	public int calcularSalario() { //sin override no sobreescribe
		return 20*horas;
	}
	
	public static void main(String[] args) {
		EmpleadoFjo ef = new EmpleadoFjo("Alejando", 80);
		System.out.println(ef.nombre);
		System.out.println(Persona.numPersonas); //numPersonas es static
		System.out.println(ef.calcularSalario());
		ef.descansar();
	}
}
